features/core_min_bootstrap.feature

```diff
@@ -3,9 +3,9 @@
 
 Feature: register a salt-minion via bootstrap
 
-  Scenario: Create bootstrap-repo for sle12sp2
+  Scenario: Create bootstrap-repo for a salt client
      Given I am authorized
-     And  I run "mgr-create-bootstrap-repo -c SLE-12-SP2-x86_64" on "server" without error control
+     And I create the bootstrap-repo for "sle-minion" on the "server"
```

Comment:
@ncounter maybe we should call it 

``` And (/^I create the x64 bootstrap-repo for "([^"]*)" on the "([^"]*)"$/) do |target|```  

because we are assuming intel. but we could have also other arch maybe in future.